src/works/work-getters.ts

```diff
@@ -191,6 +191,11 @@ export const getWorkLocked = ($workPage: WorkPage) => {
   return !!$workPage("#signin > .heading").text();
 };
 
+export const getWorkAdult = ($workPage: WorkPage): boolean => {
+  const adultCategories: WorkRatings[] = [WorkRatings.EXPLICIT, WorkRatings.MATURE, WorkRatings.NOT_RATED];
```

Comment:
since this is a constant, we should define it outside this function like this:

```ts
const ADULT_CATEGORIES: WorkRatings[] = [WorkRatings.EXPLICIT, WorkRatings.MATURE, WorkRatings.NOT_RATED];
export const getWorkAdult = ($workPage: WorkPage): boolean => {
// rest of the code
```

see https://google.github.io/styleguide/tsguide.html#identifiers-constants (but it's a common convention)